src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args)  {
```

Comment:
I think you may have accidentally added an extra spacing before `{`. Do take note that it counts towards coding standards!